lib/rails2_ruby2/action_view.rb

```diff
@@ -0,0 +1,3 @@
+# Add extensions to action_view/ directory
+
+# Shim for action_view extensions
```

Comment:
Example of a shim for components that don't have extensions today, but may in the future.